ansible.cfg

```diff
@@ -1,6 +1,7 @@
 [defaults]
 roles_path = vendor/roles
 force_handlers = True
+vault_password_file = .vault_pass.example
```

Comment:
The playbook fails if `vault_password_file` refers to a missing file. The file `.vault_pass` will be missing by default because it is `.gitignore`d. Users who don't want ansible vault would have to remove this line or create the `.vault_pass` file. I explain below why I didn't just omit this line and require vault users to add it.

To enable vault, we could have users create new stuff: add `vault_password_file` line to `ansible.cfg` and create the `.vault_pass` file. I chose the alternative of having users edit example versions of each (steps 1 and 2 in this PR's first comment) because I speculate that users would be more likely to try vault if they could edit example stuff instead of create new stuff.

An added bonus of having the examples present in the repo is that one can actually skip steps 1 and 2, and do only step 3 (run the `ansible-vault encrypt` command). That means one can test how easy it is to use ansible vault by just running that single command.